esp/src/src/nls/es/hpcc.ts

```diff
@@ -338,8 +343,15 @@ export = {
     FromTime: "Tiempo de Comenzar",
     FullName: "Nombre Entero",
     Generate: "Generar",
+    GetDFSCSV: "CSV de DFS",
+    GetDFSMap: "Mapa DFS",
+    GetDFSParents: "Padres de DFS",
+    GetLogicalFile: "Archivo lógico",
+    GetLogicalFilePart: "Parte del archivo lógico",
     GetPart: "Obtener parte",
+    GetProtectedList: "Lista Protegida",
```

Comment:
As is, it translates to Protected List, if English version is Get Protected List, this should be Obtener Lista Protegida